app/models/volunteer_op.rb

```diff
@@ -3,10 +3,18 @@ class VolunteerOp < ActiveRecord::Base
   validates :title, :description, presence: true
   validates :organisation_id, presence: true, if: "source == 'local'"
   belongs_to :organisation
-
+  
+  geocoded_by :full_address
+  after_validation :geocode, if: :different_address?   
+  after_validation :clear_lat_lng
```

Comment:
so we need this to remove incorrect lat/lng settings when the address is incomplete?

I wonder if we could make the intention of that more explicit somehow ... or is it really necessary functionality? Do we have a test wrapping it?  Did we actually encounter a problem in development?